Pipeline_example/nf1_ic.sh

```diff
@@ -0,0 +1,13 @@
+# Taken from Jenna's script from the Cellpainting repo
+
+#!/bin/bash
+
+# initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
```

Comment:
```suggestion
# initialize the correct shell for your machine to allow conda to work
```
Recommend removing the section regarding the README unless you have a section on this added in your own README for this section.

Reply:
Got it, thank you!